server/db-pg/model.js

```diff
@@ -52,13 +52,14 @@ const updateRestaurant = (id, valuesToUpdate) => {
     .join(', ');
   const queryString = `update restaurants set ${updateAssignments} where id = ${id}  returning *`;
 
-  return client.query(queryString);
+  return client.query(queryString)
+    .then(res => parseRestaurant(res.rows[0]));
 };
 
 const postRestaurant = (restaurant) => {
   stringifyRestaurant(restaurant);
   const columns = Object.keys(restaurant);
-  const values = Object.values(restaurant); // must these be in single quotes?
+  const values = Object.values(restaurant).map(value => `'${value}'`);
   const queryString = `insert into restaurants(${columns.join(', ')}) values(${values.join(', ')})`;
```

Comment:
Probably not a big deal for our purposes, but I've read that building queries with string concatenation (and I assume string templates) can open you up to injection attacks.

See here: 
https://node-postgres.com/features/queries
Relevant line is  under **Parameterized query**.

Reply:
Thanks, this is good to know! I should definitely fix this before merging.

Reply:
My bad! I'll make sure to check with you in the future.

Reply:
interesting.  Garret, what are the options? For example query string like below where text is '"'
How should it be built?
'LOAD DATA LOCAL INFILE ? INTO TABLE tableName  ' ENCLOSED BY ' +  `'${text}'`